src/main/java/ua/com/conductor/dao/MovieDao.java

```diff
@@ -0,0 +1,10 @@
+package ua.com.conductor.dao;
+
+import java.util.List;
+import ua.com.conductor.model.Movie;
+
+public interface MovieDao {
```

Comment:
suggestion: in future we will have multiple Dao, as well as multiple Services. I think it's better to start using Generic Interface principle

Reply:
For this part of hw we don't need generic dao or service. Bohdan says it today.